projects/admin/src/app/record/detail-view/holding-detail-view/serial-holding-detail-view/serial-holding-detail-view.component.html

```diff
@@ -125,7 +125,7 @@ <h1 class="mb-2" translate>Holdings</h1>
   <!-- END OF DETAILS TAB ---------------------------------------------------->
 
   <!-- LOCAL FIELDS TAB ------------------------------------------------------------>
-  <tab *ngIf="showhideLocalFieldsTab" id="holding-local-field-tab" tabOrder="3">
+  <tab *ngIf="displayLocalFieldsTab" id="holding-local-field-tab" tabOrder="3">
```

Comment:
[permissions]="[PERMISSIONS.LOFI_SEARCH, PERMISSIONS.LOFI_CREATE]"

Remove `ngIf` and use `permissions` directive